Metaverse/Source/Metaverse/Private/Components/DefiWalletCoreComponent.cpp

```diff
@@ -26,8 +26,12 @@ void UDefiWalletCoreComponent::Setup() {
     UE_LOG(LogTemp, Display, TEXT("%s was found"), *DefiWalletCore->GetActorNameOrLabel());
   } else {
     ADefiWalletCoreActor *NewDefiWalletCore = (ADefiWalletCoreActor *)GetWorld()->SpawnActor(ADefiWalletCoreActor::StaticClass());
+    // NewDefiWalletCore->myCronosChainID = 1;
+    // NewDefiWalletCore->myCronosRpc = "https://api.infura.io/v1/jsonrpc/mainnet";
     NewDefiWalletCore->myCronosChainID = 25;
     NewDefiWalletCore->myCronosRpc = "https://evm-dev.cronos.org/";
+    // NewDefiWalletCore->myCronosChainID = 338;
```

Comment:
remove comment?

Reply:
it is not comment, it is for easy testing, change chain by just uncomment them.